tst/testinstall/compressed.tst

```diff
@@ -0,0 +1,169 @@
+#@local dir,fname,isGzippedFile,stream,str
+gap> START_TEST("compressed.tst");
+gap> dir := DirectoryTemporary();;
+gap> fname := Filename(dir, "test.g.gz");;
+
+# Let us check when we have written a compressed file by checking the gzip header
+gap> isGzippedFile := function(dir, name)
+>    local out, str,prog;
+>    str := "";
+>    out := OutputTextString(str, true);
+>    Process(dir, Filename(DirectoriesSystemPrograms(),"cat"), InputTextNone(), out, [name]);
```

Comment:
Hrm, that's yucky. Can you please at least add a comment that you are not simply using `StringFile` here because that would decompress the data?

I guess what we'd need to make this less ugly is an `InputRawFile` or so which does not perform any transparent decompression. Or perhaps an optional flag to `InputTextFile` which allows turning off the decompression?

Reply:
Yes, I didn't like this. I'm happy with either of those suggestions, if anyone has a strong prference.

Reply:
Thinking a little more, the problem with introducing some kind of InputRawFile, or anything else, is it is another thing to document which (I think) no-one is ever going to want to use.

We could use the IO package to grab raw access, or introduce some undocumented kernel function to get raw access?

Reply:
Fine by me; see comments above for some ideas.

Reply:
Just to be clear: It's also OK by me to merge this now as-is; we can always revisit this later. Perhaps amend 1-2 of the minor optional remarks I made, if you feel like it; otherwise just merge it as-is *shrug*